nservicebus/scaling.md

```diff
@@ -4,38 +4,36 @@ summary: NServiceBus provides several options to scale out a system
 redirects:
  - transports/scale-out
  - nservicebus/architecture/scaling
-reviewed: 2021-07-23
+reviewed: 2024-06-17
 ---
 
-This page describes how to scale out endpoints using NServiceBus. There are several reasons to scale out, such as to achieve higher message throughput, or to provide high availability.
-
-It is important to distinguish between a logical endpoint and an endpoint instance. Review the documentation on [logical endpoints](/nservicebus/endpoints/) for more details.
+There are several reasons to scale out [logical endpoints](/nservicebus/endpoints/), such as to achieve higher message throughput, or to provide high availability to name a few.
 
 ## Splitting message handlers
 
 If message throughput is an issue, the first method to consider is splitting up [message handlers](/nservicebus/handlers/) and [sagas](/nservicebus/sagas/) over multiple logical endpoints.
 
 One message type might take considerably longer to process than other message types. The faster processing messages might suffer in throughput because of the slower processing messages. A good way to monitor and detect this is by using [ServicePulse's monitoring capabilities](/monitoring/metrics/in-servicepulse.md).
 
-Separating slower messages from faster messages leads to higher throughput for the faster messages. For this reason it can be beneficial to include messages and/or handlers in separate assemblies, making it easier to separate them from others.
+Separating slower messages from faster messages leads to higher throughput for the faster messages. For this reason, it can be beneficial to include messages and/or handlers in separate assemblies, making it easier to separate them from others.
```

Comment:
> For this reason, it can be beneficial to include messages and/or handlers in separate assemblies, making it easier to separate them from others.

I have a hard time following this reasoning.